### PR TITLE
Added new stations for Poland

### PR DESCRIPTION
**Pull request by Triloworld**
Local and official stations in Poland. Updated one station

### CODE REVIEW

File: channels/pl.m3u [hunk 1]
```diff
@@ -1,53 +1,79 @@
...
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Polish" tvg-logo="" group-title="",TV Kujawy
+http://stream.tvkujawy.pl:8080/live/broadcast.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Polish" tvg-logo="" group-title="",TV Regionalna
 https://tvreg.klemit.net/regionalna/stream/index.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Polish" tvg-logo="" group-title="",TVP3 Białystok
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Polish" tvg-logo="" group-title="",TVP Info
+http://hbbtvlive.v3.tvp.pl/hbbtvlive/livestream.php?app_id=tvpinfo
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Polish" tvg-logo="" group-title="",TVP Parlament
+http://hbbtvlive3.v3.tvp.pl/hbbtvlive/livestream.php?app_id=kanal1
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Polish" tvg-logo="" group-title="TVP3",TVP3 Białystok
```
Comment (themak1985) [inline channels/pl.m3u:32]: group title needs to be blank or ideally one of the categories we use.

File: channels/pl.m3u [hunk 2]
```diff
@@ -1,53 +1,79 @@
...
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Polish" tvg-logo="https://i.imgur.com/c59VWgF.jpg" group-title="Music",4FUN TV
 https://stream.4fun.tv:8888/hls/4f_high/index.m3u8
+#EXTINF:-1 tvg-id="WTK PL" tvg-name="" tvg-language="Polish" tvg-logo="" group-title="",dlaCiebieTV
+http://94.246.128.53:1935/tv/_definst_/dlaCiebieTv/playlist.m3u8
+#EXTINF:-1 tvg-id="WTK PL" tvg-name="" tvg-language="Polish" tvg-logo="" group-title="",Echo24
+https://insysgo.live.e57-po.insyscd.net/echo24.720.smil/chunklist_b4696000_slpl.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="" tvg-logo="https://i.imgur.com/hJAoLIq.png" group-title="",EZO TV
 http://live.ezotv.pl:1935/live/EZOTV/chunklist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="" tvg-logo="" group-title="",Sfera TV
+http://stream.sferatv.pl:1935/sferalive/smil:sferalive.smil/chunklist_w1591103044_b1128000_slpl.m3u8
```
Comment (themak1985) [inline channels/pl.m3u:11]: If possible to create an adaptive link for this and echo24, would be preferred, but this is okay.

**Review by themak1985 — COMMENTED** [review]
Checked all the new links, they all work. Request to change a couple to adaptive links if possible, though not required. What is required through is removing the group-title you added with one of the categories we use or leave blank. tvg-id should also be blank unless there is a tv guide for poland.

File: channels/pl.m3u (hunk 2 above)
Comment (Triloworld) [reply channels/pl.m3u:11]: Yes, I will fix that by using better one grabbed from website: 
http://stream.sferatv.pl:1935/sferalive/smil:sferalive.smil/playlist.m3u8

File: channels/pl.m3u (hunk 1 above)
Comment (Triloworld) [reply channels/pl.m3u:32]: Ok. I wasn't understand that feature. I will clear that text.

File: channels/pl.m3u (hunk 2 above)
Comment (Triloworld) [reply channels/pl.m3u:11]: Can you help? it don't work somehow ;/ It work with chunk one. I have fealing that chunk file won't work in future

**Comment by Triloworld** [comment]
Pleas check it once more as they won't be updated too soon.

**Comment by themak1985** [comment]
> What ID correspond to? "unless there is a tv guide for poland." Probably exist but don't how I can spot It to fix that.
> Groups are "religion" for example? Is this "category"? I will re-check if that correct

Yes, group-id are the [categories](https://github.com/iptv-org/iptv#playlists-by-category) we have.  `group-id=""` is fine. 
Poland does not have a tv guide for our repo, so `tvg-id=""` and `tvg-name=""` is fine.

**Pull request merged**